docs/structured-streaming-state-data-source.md

```diff
@@ -344,6 +368,11 @@ Each row in the source has the following schema:
   <td>int</td>
   <td>The maximum batch ID available for querying state. The value could be invalid if the streaming query taking the checkpoint is running, as the query will commit further batches.</td>
 </tr>
+<tr>
+  <td>operatorProperties</td>
+  <td>string</td>
+  <td>List of properties used by the operator encoded as JSON. Available only for operators using schema format version 2.</td>
```

Comment:
Let's not expose the format version - we can just say "operator dependent".

Reply:
Done